acceptance/suites/pre_suite/foss/10_install_release_repos.rb

```diff
@@ -1,5 +1,8 @@
 step "Setup Puppet Labs Release repositories." do
   hosts.each do |host|
-    install_puppetlabs_release_repo host
+    platform = host.platform
```

Comment:
Reading the documentation, looks like this is possible to do with `confine_block`.  `confine` statefully operates on the entire test case instance so I don't think it would be suitable here because we're doing things with different confinements in the same file.